src/openms/include/OpenMS/FORMAT/PepXMLFile.h

```diff
@@ -149,6 +149,12 @@ namespace OpenMS
     */
     void matchModification_(const double mass, const String& origin, String& modification_description);
 
+    /**
+        @brief create a TPP compatible string of the modified sequence using bracket notation
+        i.e.: n[35]RQLNK[162]LQHK[162]GEA
+    */  
+    static String modifiedSequenceToBracketString_(const AASequence & seq);
```

Comment:
I think this is never parsed by us from pepXML files and not required to determine the mod state but I can check this again. Given that there is no real standardized format for this string (at least that is how I understood it) I would only use it for writing out pepXML and not expose it to the API.